pkl-core/src/main/java/org/pkl/core/runtime/VmLanguage.java

```diff
@@ -114,4 +114,10 @@ void initializeModule(
     moduleNode.getCallTarget().call(emptyModule, emptyModule);
     MinPklVersionChecker.check(emptyModule, importNode);
   }
+
+  @Override
+  protected boolean patchContext(VmContext context, Env newEnv) {
+    // no-op
+    return true;
+  }
```

Comment:
This method is now required to use shared/pre-initialized contexts in GraalVM. It is meant to inform a hot context of a new `Env` it should use. I've implemented it as a no-op because I don't see any use of `Env` anyway.